tests/playwright-test/fixture-errors.spec.ts

```diff
@@ -435,3 +435,23 @@ test('should not teardown when setup times out', async ({ runInlineTest }) => {
   expect(result.output.split('\n').filter(line => line.startsWith('%%'))).toEqual([
   ]);
 });
+
+test('should not report fixture teardown error twice', async ({ runInlineTest }) => {
```

Comment:
Can you bring over the other test, too? 

https://github.com/microsoft/playwright/pull/11342/files#diff-7baf8520adc91e87042c746455f6e8a73b78771e9ca03095c9b15437894a6b4aR107

I intentionally had two tests so I ensured the positive test case actually does work alongside the negative test case. (Afterall, the positive case is what lead us here 😀 so I'd want to know if we broke that in the future, and it feels different enough from the other attachment cases since it happens in fixture teardown as opposed to during the test.)

Reply:
Sure, why not.